nodestream/pipeline/pipeline_file_loader.py

```diff
@@ -37,12 +47,27 @@ def configure(cls):
         cls.was_configured = True
 
     @classmethod
-    def load_file_by_path(cls, file_path: str):
-        PipelineFileSafeLoader.configure()
+    def load_file_by_path(cls, file_path: str, config: ScopeConfig = None):
+        PipelineFileSafeLoader.configure(config)
         with open(file_path) as fp:
             return load(fp, cls)
 
 
+class NodestreamProjectFileSafeLoader(SafeLoader):
```

Comment:
Should we move this to the `project` package? Its nice to have the file loaders in the same place. What do you think?